docs/man/rpmdeps.1.scd

```diff
@@ -1,50 +1,76 @@
 RPMDEPS(1)
 
 # NAME
-
 rpmdeps - Generate RPM Package Dependencies
 
 # SYNOPSIS
+*rpmdeps* [options]
 
-*rpmdeps* [options] {*-P*|*--provides*} {*-R*|*--requires*} [_FILE_ ...]
+*rpmdeps* [options] _FILE_ ...
 
 # DESCRIPTION
+*rpmdeps* generates package dependencies for one or more files, received
+either as a list of filenames on the standard input, or command line
+arguments.
+
+*rpmdeps* is mainly useful for troubleshooting dependency generation
+and developing new dependency generators.
+
+Note that it is not actually used by *rpmbuild*(1) for automatic
+dependency generation except in a legacy compatibility mode.
 
-*rpmdeps* generates package dependencies for the set of _FILE_
-arguments. Each _FILE_ argument is searched for Elf32/Elf64, script
-interpreter, or per-script dependencies, and the dependencies are
-printed to stdout.
+# ARGUMENTS
+_FILE_
+	Any regular file.
 
 # OPTIONS
 See *rpm-common*(8) for the options common to all operations.
 
-*-P, --provides*
-	Print the provides
+*--alldeps*
+	Print all the dependencies.
 
-*-R, --requires*
-	Print the requires
+*--conflicts*
+	Print the conflicts dependencies.
+
+*--enhances*
+	Print the enhances dependencies.
+
+*--obsoletes*
+	Print the obsoletes dependencies.
+
+*--orderwithrequires*
+	Print the orderwithrequires dependencies.
+
+*-P, --provides*
```

Comment:
This (and -R) are actually operations and thus should be in a separate OPERATIONS section, no?

Reply:
These, along with all the --conflicts and --suggests and whatnot are actually implemented as options. Although, printing eg --provides and --requires simultaneously without any way to differentiate between them makes no sense whatsoever. But then, --alldeps is really implemented as a separate operation. I did consider this but ended up swiping it under the carpet, but you spotted the lump in it :laughing: 

In reality I think we should indeed *document* these as operations and then we can just consider the implementation buggy in that it currently allows multiple choices for a bunch of options.